docs/topics/testing/overview.txt

```diff
@@ -331,7 +331,8 @@ but it's pretty intuitive. You can consult the documentation of Python's
 :mod:`unittest` library for details.
 
 Note that the return code for the test-runner script is 1 for any number of
-failed and erroneous tests. If all the tests pass, the return code is 0. This
+failed tests (whether the failure was caused by an error, a failed assertion or
+an unexpected success). If all the tests pass, the return code is 0. This
```

Comment:
Comma is missing before 'or'/'and' (see https://en.wikipedia.org/wiki/Serial_comma).
```suggestion
failed tests (whether the failure was caused by an error, a failed assertion,
or an unexpected success). If all the tests pass, the return code is 0. This
```

Reply:
I would add `versionchanged` annotation, e.g.:
```
.. versionchanged: 4.1

    In older versions, Django test runner does not take unexpected successes,
    from tests marked with the `expectedFailure()` decorator, into account in
    the return code.

.. _speeding-up-tests-auth-hashers:
```